flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why this restruction?

Reply:
I'm not sure that app will run on aarch64

Reply:
it's clear that we'll never know.